src/DataViews/DataView.cpp

```diff
@@ -228,12 +228,14 @@ void DataView::OnEnableFrameTrackRequested(const std::vector<int>& selection) {
   metrics_uploader_->SendLogEvent(
       orbit_metrics_uploader::OrbitLogEvent::ORBIT_FRAME_TRACK_ENABLE_CLICKED);
 
+  ORBIT_CHECK(app_->HasCaptureData());
```

Comment:
This check crashes when doing the following:
1. Connect to instance and start session
2. Load Symbols for a module
3. In the function list, on a whatever function -> context menu, enable frame track

Reply:
I agree with Anton but here I want to explain when this line crashes in a different way.

Currently we can invoke the "Enable Frame Track(s)" action from the context menu in both functions tab and the live functions tab.
If we can enable frame track from the functions tab, then the capture is connected.   
If we can enable frame track from the live functions tab, then we have capture data and can check whether the capture is connected via `app_->IsCaptureConnected(app_->GetCaptureData())`.

The above code crashes when we want to enable the frame track from the functions tab but there is no capture data (e.g., we connect to the stadia instance and haven't take our first capture yet).

Reply:
A quick solution here is to show the tick / remove the tick only when hook / unhook action is enabled from the context menu. Hence, we can use `GetActionStatus`

Reply:
@akopich and I had a bit of a discussion offline. `GetActionStatus` is an interesting idea, but if I see it right, it also uses `IsCaptureConnected` to make distinctions. The biggest thing that is not nice here is that we dont have a function `IsOrbitConnected`. This would be the best option. `IsCaptureConnected` does not work properly anymore since we are opening Captures always in a new window. It also does not handle connection interruptions, like the process stopping or OrbitService crashing

Reply:
SGTM. And we should also update the way we determine the action status for hooking and unhooking to keep consistent in the future.